pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_search_modules.adoc

```diff
@@ -0,0 +1,20 @@
+[id="search-modules"]
+
+= How to search for Modules
+
+You can search for modules in Pantheon using the following tools:
+
+* A free text or keyword search in the search bar (case sensitive).
+* Filtering the search by one or more of the available filter drop-down lists.
+** Product category
+** Product version number
+** Module type (Concept, Procedure, or Reference)
+** Upload date
+
+The search results are presented in a table that displays:
+
+* The module title
+* Publication state - published or not published
+* The date the draft was uploaded
+* Module type.
```

Comment:
Periods are used for these two bullet points, but not for the others. We should be consistent one way or the other. :)